src/cpp/shared_core/Logger.cpp

```diff
@@ -47,6 +47,52 @@ namespace log {
 typedef std::map<std::string, std::shared_ptr<ILogDestination>> LogMap;
 typedef std::map<std::string, LogMap> SectionLogMap;
 
+LogLevel logLevelFromStr(const std::string& in_str)
+{
+   if (in_str == "ERROR")
```

Comment:
Looks like the code that's currently using it is parsing it from a log file so presumably spaces will be removed there.  Same with how it will be used in workbench. I'm happy to add that code though if you'd prefer. It does make life easier for the caller.